pkg/cli/cmd/bicep/publish/publish.go

```diff
@@ -94,6 +95,7 @@ type Runner struct {
 
 	File          string
 	Target        string
+	PlainHttp     bool
```

Comment:
Go style would be name this field like `PlainHTTP`. Initialisms should be capitalized.